### ISSUE TITLE
How to visualize the gml file?

### ISSUE THREAD

**Issue by youssefavx** [davidmcclure/textplot#4]
Thanks for making this cool tool! I tried it on my corpus but not sure how to visualize the gml file. I really liked your visualization and want to do something similar if possible.

**Comment by simon-20** [comment]
Agreed, it would have been great if it were explained how to produce the visualisation that it shown in the tool's README.
